deeplake/core/vectorstore/vector_search/utils.py

```diff
@@ -559,4 +559,4 @@ def is_embedding_tensor(tensor):
 
 def index_used(exec_option):
     """Check if the index is used for the exec_option"""
-    return exec_option in ("tensor_db", "compute_engine")
+    return exec_option == "compute_engine"
```

Comment:
@adolkhan I don't think this is correctly. 

Currently, in the managed DB, the index is created on the client side, since the managed DB does not support writes yet. 

This should me changed only when the managed DB support writes.

Reply:
Are we fine with requesting all people to install libdeeplake if say we change the default index thereshold from -1 to some value? Also there is inconsistency between index metrics and deep memory distance metric, we can think of some clever way to combine them, but I still believe that it should be turned off for now, and implemented directly on a managed side. Did we do benchmarking on managed db + indexing, does it give a lot of boost performance wise?

Reply:
@adolkhan These are good questions, but I think it's outside the scope of my comment. 

The index is created on the client, even for the managed DB, so the above logic can't be there. Am I missing some other consideration?